### PR TITLE
Test of Python <--> JS interface

### PR DESCRIPTION
**Pull request by giangiac**

### CODE REVIEW

File: ggg_frontend/src/templates/index.js [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+			// Create a qubit
+			const pos = new Position(x*gridSize, y*gridSize, qubitRadius-2);
+			workspace.addChild(pos);
+		}
+	}
+	const num_background_children = workspace.children.length;
+
+/////////////////////////////////////////////////////////////
+
+	const testButton = button('test of API to interface with API', gridSize, 1*gridSize, 'orange', 'black');
```
Comment (giangiac) [inline ggg_frontend/src/templates/index.js]: Here are the buttons to 'get' and 'post' using the interface.

**Comment by Gistbatch** [comment]
Hi Gian, Here is the good news: it's already working correctly!
[Screencast from 20.06.2024 13:46:47.webm](https://github.com/giangiac/tqec/assets/18579069/bdd53392-16f4-4f84-b5bf-63bce465e99d)

What are you seeing:
- First, I start the Flask server in the backend `python web/main.py`
- Then I press the get button in the interface
- In the browser console, you can see the status `200`, which means OK, and the value at the corresponding address `localhost/example`, which is `42`.
- In the terminal, you see the corresponding action: `Sending: {'name': 'example', 'value': 42}
`
- After I press the post button which is supposed to send a json object `{name: 'post_example', value: '3.14'}` to the same location. 
- In the browser console, you again retrieve the status `200 OK`.
- On the terminal, you see the corresponding action: `Received: {'name': 'post_example', 'value': '3.14'}
`

If you have questions about what the code does, please let me know.

**Comment by giangiac** [comment]
> Hi Gian, Here is the good news: it's already working correctly! [Screencast from 20.06.2024 13:46:47.webm](https://github.com/giangiac/tqec/assets/18579069/bdd53392-16f4-4f84-b5bf-63bce465e99d)
> 
> What are you seeing:
> 
> * First, I start the Flask server in the backend `python web/main.py`
> * Then I press the get button in the interface
> * In the browser console, you can see the status `200`, which means OK, and the value at the corresponding address `localhost/example`, which is `42`.
> * In the terminal, you see the corresponding action: `Sending: {'name': 'example', 'value': 42} `
> * After I press the post button which is supposed to send a json object `{name: 'post_example', value: '3.14'}` to the same location.
> * In the browser console, you again retrieve the status `200 OK`.
> * On the terminal, you see the corresponding action: `Received: {'name': 'post_example', 'value': '3.14'} `
> 
> If you have questions about what the code does, please let me know.

Thanks a lot for the breakdown of the actions!

**Comment by giangiac** [comment]
This was an utility PR to facilitate the comparison between two development snapshots which differed only on the relevant feature.

The issue has been solved and this PR has outlived its utility.

**Pull request closed without merging**